src/in/shick/diode/comments/CommentsListActivity.java

```diff
@@ -1473,6 +1473,7 @@ public void onCreateContextMenu(ContextMenu menu, View v, ContextMenuInfo menuIn
 
         if (rowId == 0) {
             menu.add(0, Constants.SHARE_CONTEXT_ITEM, Menu.NONE, "Share");
+            menu.add(0, Constants.SHARE_COMMENTS, Menu.NONE, "Share Comments");
```

Comment:
I know the codebase is horribly inconsistent about this, but it's better to put this string into strings.xml (https://github.com/zagaberoo/diode/blob/master/res/values/strings.xml) and just reference it's R.string constant here.